bootstrap.inc.sh

```diff
@@ -77,7 +80,7 @@ function _bootstrap_echo() {
 #
 function bootstrap_configure() {
   _bootstrap_echo "Bootstrap starting"
-  _bootstrap_download bootstrap.inc.sh "$BOOTSTRAP"
+   _bootstrap_download bootstrap.inc.sh "$BOOTSTRAP"
```

Comment:
why the extra space?

Reply:
I could say it was to see if you were paying attention, but the reality is probably that I wasn't paying attention. Will fix.